dense_tiles/dense_tiles.cpp

```diff
@@ -139,6 +159,8 @@ int main(int argc, char* argv[]) {
     for (unsigned int i = 0; i < grid.size(); ++i) {
         if (grid[i]) {
             sorter.emplace_back(grid[i], i);
+        } else {
+            sorter.emplace_back(0, i);
```

Comment:
If the else clause missed, tiles without any nodes would not be printed. I had a use case in my master thesis where it is useful to have a list of all tiles—even the empty ones.

EDIT:  Thank you for your comment, these lines currently look really stupid. The else statement could be replace by `else if (min_nodes == 0)` which keeps the program as quick as before for most uses cases (except printing out all tiles).

I will add a comment in the source code which explains it.

I will set the default value of min_nodes to 1 in order to keep the current behaviour and be backward compatible.